src/index.js

```diff
@@ -0,0 +1,9 @@
+module.exports.say = function say(name) 
+{
+    let innerName = "world";
+
+    if (typeof name !== "undefined") 
+        innerName = name;
+
+    return ["Hello ", innerName, "!"].join("");
```

Comment:
This is fine!

I'll approve this PR but I also suggest you to give a read to [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) article :+1: